test/aws.integration.js

```diff
@@ -163,7 +165,21 @@ describe('AWS Integration Test', () => {
 
     // eslint-disable-next-line no-console
     console.log('invoking w/o token should fail');
-    let ret = await fetch('https://lqmig3v5eb.execute-api.us-east-1.amazonaws.com/simple-package/simple-name/ci/foo');
+    let ret;
+
+    for (let tries = 3; tries >= 0; tries -= 1) {
+      // eslint-disable-next-line no-await-in-loop
+      ret = await fetch('https://lqmig3v5eb.execute-api.us-east-1.amazonaws.com/simple-package/simple-name/ci/foo');
+      if (ret.status !== 401) {
+        // eslint-disable-next-line no-console
+        console.log(`!!: ${ret.status} !== 401 (retry)`);
```

Comment:
reusing an route+integration might lead to requests initially bypassing the authorizer